### DISCUSSION TITLE
Suppression de formulaire

### DISCUSSION THREAD

**Discussion by MJMESS** [MTESSDev/FRW#54] [Annonces]
Bonjour, nous avons déployés plusieurs formulaires, mais certains d'entre eux étaient des essais et ne seront pas utilisés, est-ce qu'il y a un moyen de les supprimer afin d'épurer ?

Je ne parle pas de supprimer le formulaire d'un individu mais de supprimer le dossier avec le fichier form.yml, bind.yml et transmission.yml de certain formulaire afin qu'ils n'apparaissent plus dans l'outil pour les essais.

Merci à l'avance,

**Comment by anisite** [comment]
Bonjour, il n'est pas possible, actuellement, de supprimer de répertoire et/ou de formulaire. Il est possible cependant de réutiliser les même noms de formulaire pour effectuer un nouveau test.

Si c'est un besoin, convertissez la demande en `issue`. Nous ajouterons un fichier de `nettoyage` pour permettre de nettoyer au déploiement.

**Reply by atifmohamed** [reply]
Bonjour,
Ce même besoin existe pour mon équipe, mais il n'est pas urgent.
Est-ce que cette demande est déjà prise en charge ? 

Merci.

**Reply by anisite** [reply]
Bonsoir, non aucun `issue` n'a été ouvert à ce jour à ce sujet, nous préférons que les issues soient ouverts par les clients parce que à ce moment là vous pouvez suivre l'évolution de la demande et il est possible de vous faire valider certaines décisions.

**Reply by atifmohamed** [reply]
C'est parfait, un issue est ouvert pour ce besoin.

**Comment by j0ber** [comment]
Sera traité dans l'issue #64
